libnm-core/nm-setting-wpan.c

```diff
@@ -270,5 +357,33 @@ nm_setting_wpan_class_init (NMSettingWpanClass *klass)
 	                            G_PARAM_READWRITE |
 	                            G_PARAM_STATIC_STRINGS));
 
+	/**
+	 * NMSettingWpan:page:
+	 *
+	 * IEEE 802.15.4 channel page.
+	 **/
+	g_object_class_install_property
+	        (object_class, PROP_PAGE,
+	         g_param_spec_int (NM_SETTING_WPAN_PAGE, "", "",
+	                            NM_SETTING_WPAN_PAGE_DEFAULT,
+		                    IEEE802154_MAX_PAGE,
+		                    NM_SETTING_WPAN_PAGE_DEFAULT,
+	                            G_PARAM_READWRITE |
+	                            G_PARAM_STATIC_STRINGS));
+
+	/**
+	 * NMSettingWpan:channel:
+	 *
+	 * IEEE 802.15.4 channel.
+	 **/
```

Comment:
`Since:`

and, describe a bit more what this is, that it needs to be configured together with the page, that the range is so and so, and that -1 is the default/unset.